packages/suite/src/support/trezorConnectService.ts

```diff
@@ -0,0 +1,26 @@
+import TrezorConnect from '@trezor/connect';
+
+/**
+ * TrezorConnectService
+ *
+ * A service that provides a wrapper around TrezorConnect to make it easier to test
+ * and to provide a consistent interface for interacting with TrezorConnect.
+ *
+ * This service allows you to call TrezorConnect methods and provides a way to mock
+ * those calls in tests.
+ */
+export const createTrezorConnectService = (trezorImplementation: typeof TrezorConnect) => {
+    /**
+     * Call a TrezorConnect method
+     * @param fn Function to call with TrezorConnect as the argument
+     * @returns The result of the function call
+     */
+    const invoke = <R>(fn: (trezorConnect: typeof TrezorConnect) => R): R =>
```

Comment:
Why? To not call the `TrezorConnect` statically! (we should avoid to prevent us from mamking that bizzare build steps replacement of implmentations btw)